### ISSUE TITLE
SSR hydration is not working

### ISSUE THREAD

**Issue by oysteinxyz** [mvolkmann/svelte-ssr#2]
Reproduce:

- npm i
- change name prop in server.js to something else
- npm run build:ssr
- npm run serve

It still shows the name: "world" as its the default, so SSR hydration is not working.
It might have gone unnoticed because the prop default and SSR prop was the same (name: "world").

Actually SSR is not working at all for me, looking at the chrome network tab; the response is just the template with bundle.js, no SSR.

Btw the "cp" command does not exist on windows. I replaced it with "copy" as a windows workaround.

**Comment by mvolkmann** [comment]
Thanks for reporting this! I think it's fixed. Check out the minor changes I made in the last commit. Let me know if it still doesn't work for you ... minus the cp vs. copy issue.

**Comment by oysteinxyz** [comment]
Getting 404 when serving now, so the app is not working.

![err](https://user-images.githubusercontent.com/59795195/85842139-7f33b680-b79f-11ea-94d4-d5e63a964f02.png)

And if I add

> app.use(express.static('public'));

It finds the bundle and starts the app but, the name prop is overridden to "world" no matter what you put it to in server.js

For debugging, in server.js try changing the name prop to something else. Then writing in the input field when the app is served.

**Comment by mvolkmann** [comment]
Yikes! Not sure why it seemed to be working last night. I can't get it to work now. I'm at a loss for how to fix this and could use some help. Honestly when I want SSR I use Sapper where it's built in. I just wanted to demonstrate that it is possible to implement it without using Sapper. The docs certainly claim that.

**Comment by rookbreezy** [comment]
I have been playing with the Svelte SSR API in my own project. I have got everything working except SSR props. I'm stuck there. I think it's a technical limitation with Svelte at the moment.

I can't get Sapper working with the stack I have, I also don't like how its kind of a black box and you are locked from changing rollup config.

This is one complex workaround I found: https://github.com/svelte-view-engine but does not work on Windows which I need. Something like that or Sapper seems to be the only option for SSR.
